src/config/mod.rs

```diff
@@ -5364,7 +5374,7 @@ mod tests {
             ..Default::default()
         };
         let provider = Some(MockRegionInfoProvider::new(vec![]));
-        let cf_opts = build_cf_opt!(config, CF_DEFAULT, &cache, no_limiter.as_ref(), provider);
+        let cf_opts = build_cf_opt!(config, CF_DEFAULT, &cache, no_limiter.as_ref(), provider,);
```

Comment:
The added semi comma makes this line look worse.

Reply:
Good catch. Fixed。